src/jit/compiler.h

```diff
@@ -5770,7 +5772,9 @@ class Compiler
 #define LPFLG_VAR_LIMIT 0x0100    // iterator is compared with a local var (var # found in lpVarLimit)
 #define LPFLG_CONST_LIMIT 0x0200  // iterator is compared with a constant (found in lpConstLimit)
 #define LPFLG_ARRLEN_LIMIT 0x0400 // iterator is compared with a.len or a[i].len (found in lpArrLenLimit)
-#define LPFLG_SIMD_LIMIT 0x0080   // iterator is compared with Vector<T>.Count (found in lpConstLimit)
+#define LPFLG_SIMD_LIMIT                                                                                               \
+    0x0080 // iterator is compared with Vector<T>, Vector64<T>, Vector128<T>, or Vector256<T>.Count (found in
+           // lpConstLimit)
```

Comment:
nit: This formatting looks pretty weird. Did jit-format doe this automatically? I think it would be OK if you aligned it as before, and manually split the constant to the next line.

Reply:
Yes, this is how the format patch fixed it up. I might just fix this up to say `vector count` as well to keep the comment shorter.